### PR TITLE
update readme for storybook issue

### PR DESCRIPTION
**Pull request by AllStackDev1**
Closes: #1620
# Description
There are known issues with the way Storybook integrates with SolidStart framework. As such, most of what is in Storybook are the basic reusable components. This should be explained so that consumers are aware of these limitations with the integration.

### CODE REVIEW

File: solidstart-tanstackquery-tailwind-modules/README.md [hunk 1]
```diff
@@ -33,6 +33,11 @@ SolidStart is a meta framework for SolidJS that is built on top of the React API
 - [ESLint](https://eslint.org/) - Code linting
 - [Prettier](https://prettier.io/) - Code formatting
 
+#### Note*
+We encountered an issue using Storybook in this showcase involving routes, you can follow the discord discussion [here](https://discord.com/channels/722131463138705510/1072093687892426843)
```
Comment (vyktoremario) [inline solidstart-tanstackquery-tailwind-modules/README.md:37]: I think it makes sense to give a brief summary of what the issue is before providing a link to follow the discussion.

**Review by vyktoremario — CHANGES_REQUESTED** [review]
Left a comment

File: solidstart-tanstackquery-tailwind-modules/README.md (hunk 1 above)
Comment (AllStackDev1) [reply solidstart-tanstackquery-tailwind-modules/README.md]: Ok

**Review by kenslachtajr — DISMISSED** [review]

**Review by Megio — COMMENTED** [review]
I see also a check annotation in a file unrelated to this PR, but probably we can fix that here too:

in `dynamicColor.ts` we should remove that `any` in the `getRGB` function.
we can typed that parameter like this:

```
function getRGB(c: string | number) {
  return typeof c === 'string' ? parseInt(c, 16) : c;
}
```

**Comment by AllStackDev1** [comment]
> I see also a check annotation in a file unrelated to this PR, but probably we can fix that here too:
> 
> in `dynamicColor.ts` we should remove that `any` in the `getRGB` function. we can typed that parameter like this:
> 
> ```
> function getRGB(c: string | number) {
>   return typeof c === 'string' ? parseInt(c, 16) : c;
> }
> ```

This causes another set of errors.

image

Not sure of the purpose of that function so won't want to touch it much. lol

File: solidstart-tanstackquery-tailwind-modules/README.md [hunk 2]
```diff
@@ -33,6 +33,12 @@ SolidStart is a meta framework for SolidJS that is built on top of the React API
 - [ESLint](https://eslint.org/) - Code linting
 - [Prettier](https://prettier.io/) - Code formatting
 
+#### Note*
+We encountered an issue using Storybook in this showcase involving routes `Route layout not define`.
```
Comment (honzikec) [inline solidstart-tanstackquery-tailwind-modules/README.md:37]:
```suggestion
We encountered an issue using Storybook in this showcase involving routes `Route layout not defined`.
```

**Review by honzikec — CHANGES_REQUESTED** [review]
I think I found a small typo.

**Comment by hdJerry** [comment]
```
{
    "owner": "thisdot",
    "name": "starter.dev-github-showcases",
    "branch": "main",
    "path/": "angular-apollo-tailwind/projects"
}
```

not sure why Solid Start params object have path with a "/"  suddenly.

**Review by Megio — CHANGES_REQUESTED** [review]
Nested folder navigation is broken, if you try to navigate inside a folder and then again inside another child folder, the second click doesn't work

**Review by Megio — APPROVED** [review]

**Pull request merged**